bin/reth/src/db/mod.rs

```diff
@@ -193,7 +200,32 @@ impl<'a, DB: Database> DbTool<'a, DB> {
             walker.skip(start).take(len).collect::<Vec<_>>()
         })?;
 
-        println!("{data:?}");
+        if data.is_empty() {
+            error!("Table is empty.");
+            return Ok(())
+        }
+
+        let mut table = ComfyTable::new();
+        table.load_preset(comfy_table::presets::ASCII_MARKDOWN);
+        table.set_header(["Entry Index", "Key", "Value"]);
+        for (i, entry) in data.into_iter().enumerate() {
+            match entry {
+                Ok((key, value)) => {
+                    let mut row = Row::new();
+                    row.add_cell(Cell::new(format!("{}", start + i)))
```

Comment:
Smol nit: I don't think we really need this index as it doesn't mean much to us outside of this context. Perhaps a better format would be to print something like "Showing entries {start}..{end} out of {entries} entries" so we can use that to navigate?